sdk/identity/identity/README.md

```diff
@@ -2,15 +2,7 @@
 
 The Azure Identity library provides Azure Active Directory token authentication support across the Azure SDK. It provides a set of [TokenCredential](https://docs.microsoft.com/javascript/api/@azure/core-auth/tokencredential) implementations which can be used to construct Azure SDK clients which support AAD token authentication.
 
-This library currently provides credentials for:
-
-- [Service principal authentication](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals)
-- [Managed identity authentication](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview)
-- [Device code authentication](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth2-device-code)
-- Interactive browser authentication, based on OAuth2 authentication code that uses the default system browser.
-- [Username + password authentication](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth-ropc)
-- Visual Studio Code authentication, with the login information saved in Azure plugin for Visual Studio Code
-- Azure CLI authentication, with the login information saved in Azure CLI
+You can find examples for these various credentials in [Azure Identity Examples Page](https://github.com/Azure/azure-sdk-for-js/blob/hotfix/identity_1.3.0/sdk/identity/identity/samples/AzureIdentityExamples.md)
```

Comment:
This is the only addition on top of the changes ported over from #14646. I'd like to make this change in the master branch as well.

Reply:
The other change is to point to the newly added file using the hot fix tag so that future additions which might be v2 specific done to the file in master branch does not show up for v1 users